2016/proposals/patrick-kofod-mogensen.md

```diff
@@ -0,0 +1,56 @@
+# Making Optim.jl great again
```

Comment:
Well, may want to tone this down since I'm not sure how the google reviewers will like it